### PR TITLE
Create namingPet branch and all the relate files

### PR DESCRIPTION
**Pull request by Balloii**

### CODE REVIEW

File: lib/src/data/respository/pet_repository.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import 'package:flutter_cache_manager/flutter_cache_manager.dart';
+import 'package:http/http.dart' as http;
+import 'package:jitd_client/src/data/models/pet_model.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+import '../models/test_model.dart';
+
+class PetRepository {
+  final String _petNamingUrl =
+      "https://jitd-backend.onrender.com/v1/users/pet/id";
```
Comment (aofattaporn) [inline lib/src/data/respository/pet_repository.dart]: Actually, I want to  separate between local URL with  Global URL

**Review by aofattaporn — COMMENTED** [review]
I comment on one more, about URLs Name, pls to check My review and push again

**Comment by aofattaporn** [comment]
Before merging, Pls review more than 2 people @teamchz  @sura1542  @thunyathep

File: lib/src/data/respository/pet_repository.dart (hunk 1 above)
Comment (aofattaporn) [reply lib/src/data/respository/pet_repository.dart]: you can looking examples within post_repository.dart

**Review by teamchz — APPROVED** [review]

**Review by sura1542 — APPROVED** [review]

**Review by thunyathep — APPROVED** [review]

**Review by aofattaporn — APPROVED** [review]
LGTM

**Pull request merged**